patches/@openedx+frontend-build+13.0.28.patch

```diff
@@ -92,27 +52,8 @@ index 2879dd9..1ddb07f 100644
 +if (envConfigPath) {
 +  const envConfigFilename = envConfigPath.slice(envConfigPath.indexOf('env.config'));
 +  fs.copyFileSync(envConfigPath, envConfigFilename);
-+
-+  const newConfigFilepath = path.resolve(process.cwd(), envConfigFilename);
-+  envConfig = require(newConfigFilepath);
 +}
 +
  // Add process env vars. Currently used only for setting the PUBLIC_PATH.
  dotenv.config({
    path: path.resolve(process.cwd(), '.env'),
-@@ -45,12 +65,12 @@ if (process.env.ENABLE_NEW_RELIC !== 'false') {
-     agentID: process.env.NEW_RELIC_AGENT_ID || 'undefined_agent_id',
-     trustKey: process.env.NEW_RELIC_TRUST_KEY || 'undefined_trust_key',
-     licenseKey: process.env.NEW_RELIC_LICENSE_KEY || 'undefined_license_key',
--    applicationID: process.env.NEW_RELIC_APP_ID || 'undefined_application_id',
-+    applicationID: envConfig.NEW_RELIC_APP_ID || process.env.NEW_RELIC_APP_ID || 'undefined_application_id',
-   }));
-   extraPlugins.push(new NewRelicSourceMapPlugin({
--    applicationId: process.env.NEW_RELIC_APP_ID,
-+    applicationId: envConfig.NEW_RELIC_APP_ID || process.env.NEW_RELIC_APP_ID,
-     apiKey: process.env.NEW_RELIC_ADMIN_KEY,
--    staticAssetUrl: process.env.BASE_URL,
-+    staticAssetUrl: envConfig.BASE_URL || process.env.BASE_URL,
-     // upload source maps in prod builds only
-     noop: typeof process.env.NEW_RELIC_ADMIN_KEY === 'undefined',
-   }));
```

Comment:
We're going to have to remove this patch, I think.  See https://github.com/openedx/frontend-build/pull/515#issuecomment-2079430708.